src/components/board/Stack.vue

```diff
@@ -178,6 +181,11 @@ export default {
 		deleteStack(stack) {
 			this.$store.dispatch('deleteStack', stack)
 		},
+		archiveAllCardsFromStack(stack) {
+			this.cardsByStack.forEach(card => {
```

Comment:
Not fully convinced about this, we should somehow have a progress shown about this and make sure that the changes are not done until all the archive operations have been performed.